plugin-taskbar/lxqttaskbar.cpp

```diff
@@ -83,6 +86,9 @@ LXQtTaskBar::LXQtTaskBar(ILXQtPanelPlugin *plugin, QWidget *parent) :
     QTimer::singleShot(0, this, SLOT(settingsChanged()));
     setAcceptDrops(true);
 
+    connect (m_pSignalMapper, SIGNAL(mapped(int)), this, SLOT(activateTask(int)));
```

Comment:
or should I use kind of mixed?
connect (m_pSignalMapper, SIGNAL(mapped(int)), this, &LXQtTaskBar::activateTask);

Sorry I am confused...